src/test/java/org/ambraproject/rhino/service/impl/SyndicationCrudServiceImplTest.java

```diff
@@ -0,0 +1,175 @@
+import org.springframework.context.annotation.Configuration;
+import org.springframework.orm.hibernate3.HibernateTemplate;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ContextConfiguration(classes = SyndicationCrudServiceImplTest.class)
+@Configuration
+public class SyndicationCrudServiceImplTest extends AbstractStubbingArticleTest {
+
+  private SyndicationCrudService mockSyndicationCrudService;
+
+  private MessageSender mockMessageSender;
+
+  private RuntimeConfiguration mockRuntimeConfiguration;
+
+  private JournalCrudService mockJournalCrudService;
+
+  private ArticleCrudService mockArticleCrudService;
+
+  private HibernateTemplate mockHibernateTemplate;
+
+  private List<Syndication> stubSyndications = ImmutableList.of(createStubSyndication());
+
+  @Bean
+  public SyndicationCrudService syndicationCrudService() {
+    mockSyndicationCrudService = spy(SyndicationCrudServiceImpl.class);
+    LOG.debug("syndicationCrudService() * --> {}", mockSyndicationCrudService);
+    return mockSyndicationCrudService;
+  }
+
+  @Bean
+  public MessageSender messageSender() {
+    mockMessageSender = mock(MessageSender.class);
+    LOG.debug("messageSenderCrudService() * --> {}", mockMessageSender);
+    return mockMessageSender;
+  }
+
+  private Syndication createStubSyndication() {
+    return new Syndication(createStubArticleRevision(), "test");
+  }
+
+  @BeforeMethod
+  public void initMocks() throws IllegalAccessException, NoSuchFieldException {
+    mockSyndicationCrudService = applicationContext.getBean(SyndicationCrudService.class);
+    mockArticleCrudService = applicationContext.getBean(ArticleCrudService.class);
+    mockRuntimeConfiguration = applicationContext.getBean(RuntimeConfiguration.class);
+    mockJournalCrudService = applicationContext.getBean(JournalCrudService.class);
+    mockHibernateTemplate = applicationContext.getBean(HibernateTemplate.class);
+    mockMessageSender = applicationContext.getBean(MessageSender.class);
+  }
+
+  @Test
+  public void testGetSyndication() throws Exception {
+    final ArticleRevisionIdentifier revisionId = ArticleRevisionIdentifier.create("0", 1);
+    when(mockArticleCrudService.readRevision(revisionId)).thenReturn(createStubArticleRevision());
+    when(mockHibernateTemplate.execute(any())).thenReturn(createStubSyndication());
+
+    mockSyndicationCrudService.getSyndication(revisionId, "test");
+  }
+
+  @Test
+  public void testGetSyndications() throws Exception {
+    final ArticleRevisionIdentifier revisionId = ArticleRevisionIdentifier.create("0", 1);
```

Comment:
nit: When using constants as method call parameters, I usually like to add a comment to describe the parameter to clarify for the reviewer. For example:
```
revisionId = ArticleRevisionIdentifier.create("0" /* articleIdentifier */, 1 /* revision */);
```

Reply:
I used to do this myself. Not to sound like an IntelliJ shill here, but funnily enough they added this as a feature by default recently. I don't want to add it here because then there would be duplicate labels when viewing the code in IntelliJ. :man_shrugging: